src/connectors/radiostudent.ts

```diff
@@ -0,0 +1,5 @@
+export {};
+
+Connector.playerSelector = '#icecast_list';
+Connector.artistTrackSelector = '#icecast_list > ul > li > p';
+Connector.currentTimeSelector = '#icecast_list > ul > li > h3';
```

Comment:
This one is incorrect - `currentTimeSelector` is more for like when you have a player like `00:02 / 04:23`, where `00:02` would be the current play time of a track.

Reply:
Thanks for the clarification! I will remove the `currentTimeSelector`.